runtime/src/iree/builtins/ukernel/CMakeLists.txt

```diff
@@ -172,7 +172,6 @@ iree_bitcode_library(
     "${PROJECT_BINARY_DIR}/runtime/src/iree/schemas/cpu_data_headers_filegroup.stamp"
     "internal_headers_filegroup.stamp"
   SRCS
-    "fallback.c"
```

Comment:
Yes, dropping this is part of how you make room for your optimized kernel code.  (this comment is mostly a note to self - I hadn't looked at that code in a while).